piper/buttondialog.py

```diff
@@ -220,6 +220,7 @@ def _listbox_filter_func(self, row):
         return True
 
     def _get_button_name_and_description(self, button):
+        # Translators: the {} will be replaced with the button index.
```

Comment:
So how about:
`# Translators: the {} will be replaced with the button index, e.g. "Button 1 click".`